app/src/main/java/com/github/wnder/user/User.java

```diff
@@ -78,6 +79,15 @@ public Location getPositionFromGPS(LocationManager manager, Context context){
      */
     public abstract void onNewPictureAvailable(LocationManager manager, Context context, Consumer<String> pictureIdAvailable);
 
+    /**
+     * Apply a function once the designated list of pictures of the user have been retrieved
+     * @param picturesListName The name of the list of pictures to get from the databse (ex.: guessedPics, uploadedPics)
+     * @param PicsAv Function to apply
+     */
+    public void onPicturesAvailable(String picturesListName, Context ctx, Consumer<List<String>> PicsAv){
+        //TODO: add local database implementation
```

Comment:
It doesn't make the app crash? If it does you should probably just accept an empty list for now.

Reply:
It right, it has unexpected behaviour in case we have no internet connection. So I will add this change.